app/controlplane/internal/server/grpc.go

```diff
@@ -238,3 +242,12 @@ func requireRobotAccountMatcher() selector.MatchFunc {
 		return r.MatchString(operation)
 	}
 }
+
+// Matches all operations except the ones that require the organization to be set
+func requireAllButOrganizationOperationsMatcher() selector.MatchFunc {
+	const requireMatcher = "/controlplane.v1.OrganizationService/Create|/controlplane.v1.*/ListMemberships"
```

Comment:
`skipMatcher`, we can put it that name yes.

Because there are apparently several endpoints for ListMemberships in different services, from `UserService` and `OrganizationService`